example/.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [ "axway/env-titanium" ],
+  "parserOptions": {
+    "ecmaVersion": 2015,
+    "sourceType": "script"
+  },
+  "overrides": [
```

Comment:
The `overrides` option is not needed here. `dangerfile.js` is in the root directory so the `.eslintrc` there handles it.